test/jobs/somalier/test_relate.py

```diff
@@ -249,15 +246,15 @@ def test_moves_somalier_output_to_expected_batch_resource_files(
 
         cmd = get_command_str(relate_j)
         assert re.search(
-            fr'mv related.pairs.tsv \${{BATCH_TMPDIR}}/.+\/output_pairs',
+            r'mv related.pairs.tsv \${BATCH_TMPDIR}/.+\/output_pairs',
```

Comment:
Apologies to the author of this, who now has to deal with half the lines saying `${{BATCH_TMPDIR}}` and the other half saying `${BATCH_TMPDIR}`… 🙃

Reply:
placeholder in an a regex string, I don't love this haha